src/librustdoc/html/templates/sidebar.html

```diff
@@ -15,7 +15,9 @@ <h2 class="location"> {# #}
             {% for block in blocks %}
                 {% if block.should_render() %}
                     {% if !block.heading.name.is_empty() %}
-                        <h3><a href="#{{block.heading.href|safe}}">{{block.heading.name}}</a></h3>
+                        <h3><a href="#{{block.heading.href|safe}}"> {# #}
+                            {{block.heading.name|wrapped|safe}} {# #}
```

Comment:
I don't think it's useful here considering all the longest headings already have whitespace characters, making them split nicely.

Reply:
Hum I see. Then nevermind.